test/tools/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Please forgive me for this -- build AWS SDK v2 from source, see https://github.com/aws/aws-cli/issues/4685#issuecomment-1094307056
+FROM python:3.9-alpine as installer
+
+RUN set -ex; \
+    apk add --no-cache \
+    git unzip groff \
+    build-base libffi-dev cmake
+
+ENV AWS_CLI_VERSION=2.7.1
+RUN set -eux; \
+    mkdir /aws; \
+    git clone --single-branch --depth 1 -b ${AWS_CLI_VERSION} https://github.com/aws/aws-cli.git /aws; \
+    cd /aws; \
+    sed -i'' 's/PyInstaller.*/PyInstaller==4.10/g' requirements-build.txt; \
+    python -m venv venv; \
+    . venv/bin/activate; \
+    ./scripts/installers/make-exe
+
+RUN set -ex; \
+    unzip /aws/dist/awscli-exe.zip; \
+    ./aws/install --bin-dir /aws-cli-bin; \
+    /aws-cli-bin/aws --version
+
+FROM alpine:latest
+
+# Install APK packages
+RUN echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories
+RUN apk update
+RUN apk add --no-cache \
+    curl \
+    tar \
+    bash \
+    git \
+    jq \
+    make \
+    go \
+    gcompat \
+    helm@testing \
+    kubectl@testing
+
+# Install AWS CLI
+COPY --from=installer /usr/local/aws-cli/ /usr/local/aws-cli/
+COPY --from=installer /aws-cli-bin/ /usr/local/bin/
+RUN aws --version
+
+# Install eksctl
+RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
```

Comment:
nit: Can we pin this to a version? I think we tested v0.94.0 for the setup instructions. Latest is v0.105.0, and as more k8s/eksctl versions come out, older versions become unsupported.

https://github.com/weaveworks/eksctl/releases/download/v0.94.0/eksctl_Darwin_amd64.tar.gz